tools/hangar/__snapshots__/test_corpus/extern_implementation.w/compile/tf-aws/proc.104793feca8a0c99ea8c2876b3c2124f464b0b5bffc723938b058ed94174607f/index.js

```diff
@@ -0,0 +1,4 @@
+async handle() {
```

Comment:
@Chriscbr - wondering why this file was created in this hash named folder. Is there another place we should have changed?

Reply:
Just pulled the latest changes and ran hangar tests again, but no changes

Reply:
Looking into it...

Reply:
@staycoolcall911 When I look at the parent folder, there's two `proc.<blah>` folders as well as `proc1` and `proc2`. I think these folders are just leftover / you can delete them from source control and they shouldn't be regenerated by the hangar tests.

We can add this line to the `.gitignore` to prevent the issue for others:
```
tools/hangar/__snapshots__/**/*.proc/
```

Reply:
Ok, done. Thanks for looking into it

Reply:
Hmmm... @Chriscbr rerunning hangar still recreated these folders - you can give it a try on my branch:
<img width="670" alt="image" src="https://user-images.githubusercontent.com/106860404/233147657-2d47e2f5-2529-48b3-ba25-a49b21828b3b.png">

Reply:
Maybe there is a caching problem with NX? Try running `npx nx build winglang --skip-nx-cache`

Reply:
Yes - that did the trick, thanks!